pkg/api/aim/dashboards.go

```diff
@@ -5,16 +5,27 @@ import (
 
 	"github.com/gofiber/fiber/v2"
 	"github.com/google/uuid"
+	log "github.com/sirupsen/logrus"
 	"gorm.io/gorm"
 
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/api"
+	"github.com/G-Research/fasttrackml/pkg/common/middleware/namespace"
 	"github.com/G-Research/fasttrackml/pkg/database"
 )
 
 func GetDashboards(c *fiber.Ctx) error {
+	ns, err := namespace.GetNamespaceFromContext(c.Context())
+	if err != nil {
+		return api.NewInternalError("error getting namespace from context")
+	}
+	log.Debugf("getDashboards namespace: %s", ns.Code)
+
 	var dashboards []database.Dashboard
 	if err := database.DB.
+		Preload("App").
```

Comment:
This seems like more mixing of model and controller. While it was here already, maybe now is a good time to abstract dashboards and apps out? It doesn't seem like a controller responsible for rendering json should be talking to a database...

Reply:
yes, we have to refactor `aim` and make it similar to what we have in `mlflow`. right now `aim` is in bad shape.